docs/SlavaUkraini/examples/video-overlay/index.md

```diff
@@ -1,19 +1,81 @@
+title: Overlays
 ---
 
-## Video on webpages
+## Overlays
 
-Video used to be a hard task when building a webpage, until the [`<video>` HTML element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video) was made available.
+There are 3 Overlays in Leaflet API: 
+- [`ImageOverlay`](/reference.html#imageoverlay): Raster Layer, Extends [`Layer`](/reference.html#layer)
+- [`VideoOverlay`](/reference.html#videooverlay): Raster Layer, Extends [`ImageOverlay`](/reference.html#imageoverlay)
+- [`SVGOverlay`](/reference.html#svgoverlay): Vector Layer, Extends [`ImageOverlay`](/reference.html#imageoverlay)
 
-Nowadays, we can use the following HTML code:
+In this tutorial, you’ll learn how to use these Overlays.
+
+### `ImageOverlay`
+
+`L.ImageOverlay` is used to load and display a single image over specific bounds of the map. 
+
+To add an image overlay [`L.ImageOverlay`](/reference.html#imageoverlay) use this:
+```
+var overlay = L.imageOverlay( imageUrl, latLngBounds, options );
+```
+First of all, create a Leaflet map and add a background `L.TileLayer` in the usual way:
+```
+var map = L.map('map').setView([37.8, -96], 4);
+
+L.tileLayer('https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_token=' + mapboxAccessToken, {
+	maxZoom: 18,
+	attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, ' +
+		'Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
+	id: 'mapbox/satellite-v9',
+	tileSize: 512,
+	zoomOffset: -1
+}).addTo(map);
+```
+
+Let's create an image overlay with multiple options:
+```
+var imageUrl = 'https://maps.lib.utexas.edu/maps/historical/newark_nj_1922.jpg';
+var errorOverlayUrl = 'https://cdn-icons-png.flaticon.com/512/110/110686.png';
+var altText = 'Image of Newark, N.J. in 1922. Source: The University of Texas at Austin, UT Libraries Map Collection.';
+var latLngBounds = L.latLngBounds([[40.799311, -74.118464], [40.68202047785919, -74.33]]);
+
+var imageOverlay = L.imageOverlay( imageUrl, latLngBounds, {
+	opacity: 0.8,
+	errorOverlayUrl: errorOverlayUrl,
+	alt: altText,
+	interactive: true
+}).addTo(map);
+```
+If you want to see the area which is covered by the ImageOverlay, you can add a [`L.Rectangle`](/reference.html#rectangle) with the same `L.LatLngBounds` to the map:
+```
+L.rectangle(latLngBounds).addTo(map);
+map.fitBounds(latLngBounds);
+```
+- `opacity` defines the opacity of the image overlay, it equals to `1.0` by default. Decrease this value to make an image overlay transparent and to expose the underlying map layer. 
+	
+- `errorOverlayUrl` is a URL to the overlay image to show in place of the overlay that failed to load.
+
+- `alt` stands for the alt attribute of the image. Images alternatives (`alt` option) add valuable information for low vision or blind screen reader users. Image alternatives also benefit people who have poor or unstable internet, some cognitive disabilities. Moreover, it can improve the SEO of a website.
```

Comment:
```suggestion
- `alt` sets the HTML [`alt`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-alt) attribute to provide an alternative text description of the image. Alternative text is essential information for screen reader users. It can also benefit people during poor network connectivity, in the case the image fails to load. Moreover, it can improve the SEO of a website.
```

A little rewrite to detail what `alt` really is and why it is useful. I also appreciate mentioning users with cognitive disabilities, though I think it is borderline TMI in this context, and perhaps better suited in https://leafletjs.com/SlavaUkraini/examples/accessibility/.